File: js/task2b/S2_FOR_02.js
function findFirstEven(nums) {
  for (let i = 0; i < nums.length; i++) {
      if (nums[i] % 2 === 0) {
          return nums[i];
      }
  }
  return null;
}

console.log(findFirstEven([1, 3, 2, 4, 5]));  
console.log(findFirstEven([1, 3, 5]));         
console.log(findFirstEven([2, 1, 3]));       
console.log(findFirstEven([]));               

File: python/task2b/S2_FOR_02.py
def findFirstEven(nums):
  for n in nums:
      if n % 2 == 0:
          return n
  return None

print(findFirstEven([1, 3, 2, 4, 5]))
print(findFirstEven([1, 3, 5]))
print(findFirstEven([2, 1, 3]))
print(findFirstEven([]))
